packages/admin/src/App/Home/Products.js

```diff
@@ -123,7 +123,7 @@ class Products extends React.Component {
         />
         <Modal
           title="Delete Product"
-          visible={isModalOpen}
+          visible={selectedProd}
```

Comment:
Since this is not a boolean, it is showing errors in the console
You can add double negation !!selectedProduct or check if it equal to null